include/z64camera.h

```diff
@@ -27,6 +27,16 @@
 #define PARENT_CAM(cam) ((cam)->play->cameraPtrs[(cam)->parentCamId])
 #define CHILD_CAM(cam) ((cam)->play->cameraPtrs[(cam)->childCamId])
 
+// All shops and house scenes are expected to have their first two bgCamInfo entries to be the following indices:
+#define BGCAM_INDEX_TOGGLE_FIXED 0
+#define BGCAM_INDEX_TOGGLE_PIVOT 1
```

Comment:
The first two entries should be at indices 0 and 1? :thonk:

Reply:
Well there can be any number of entries in the BgCamInfo List, and these two bgCam entries have to be entries 0 and 1 respectively, they have to be in that order and they can't be further along in the list. That's what I was trying to get at

Reply:
I think alot of the confusion is fixed if you remove the word "indices". Its confusing cause the values themselves are indices, and youre talking about their index (position in the list among other values which are themselves indices). so its index-ception

`// All shops and house scenes are expected to have their first two bgCamInfo entries be the following:`